lua/acf/core/classes/grouped.lua

```diff
@@ -80,6 +94,9 @@ function Classes.GetGroup(Namespace, ID)
 	end
 end
 
+--- Indexes the groups stored in Entries into a new Namespace, with helper functions
+--- @param Namespace table The namespace to store the groups under
```

Comment:
The namespace in this case will be the table that will receive the new functions.